language/use-cases/applying-llms-to-data/bigquery_dataframes_llm_code_generation.ipynb

```diff
@@ -0,0 +1,894 @@
+{
```

Comment:
<p>Need to update links here to genai GitHub repo instead of other locations</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/201/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Links are updated.